modules/core/src/main/java/org/apache/synapse/config/xml/BeanMediatorFactory.java

```diff
@@ -74,6 +74,8 @@ public Mediator createSpecificMediator(OMElement elem, Properties properties) {
             handleException("'action' attribute of Bean mediator is required.");
         }
 
+        addAllCommentChildrenToList(elem, mediator.getCommentsList());
+
```

Comment:
Remove these new lines

Reply:
Fixed in https://github.com/wso2/wso2-synapse/pull/1439